appveyor.yml

```diff
@@ -8,6 +8,7 @@ pull_requests:
 branches:
   only:
     - master
+    - add-choco-package
```

Comment:
this second branch should be removed. It's only to run the build in this PR to test the changes in appveyor.yml itself.